src/v2/Apps/Consign/OfferDetail/Components/CreateOfferResponse.ts

```diff
@@ -0,0 +1,41 @@
+import { Environment, commitMutation, graphql } from "react-relay"
+import {
+  CreateOfferResponseMutation,
+  CreateOfferResponseMutationInput,
+  CreateOfferResponseMutationResponse,
+} from "v2/__generated__/CreateOfferResponseMutation.graphql"
+
+type CreateOfferResponseValues = Pick<
+  CreateOfferResponseMutationInput,
+  "intendedState"
+>
+
+export const CreateOfferResponse = (
```

Comment:
(thought) Unless this will eventually expand into an actual react component that returns dom nodes might be helpful to name it `CreateOfferResponseMutation` or to separate the mutation out into a `mutation` folder

Reply:
That makes sense. I don't see this being anything more than an abstraction for the mutation itself. I'll address this in this PR.